scipy/integrate/quadpack.py

```diff
@@ -633,6 +633,18 @@ def tplquad(func, a, b, gfun, hfun, qfun, rfun, args=(), epsabs=1.49e-8,
     odeint: ODE integrators
     scipy.special: For coefficients and roots of orthogonal polynomials
 
+    Examples
+    --------
+
+    Returns triple integration of ``x*y*z``
+    x ranging from 1 to 2,y ranging from 2 to 3,z ranging from 0 to 1
```

Comment:
`x` ranging from 1 to 2, `y` ranging from ...
Spaces after comma and variables in single backticks.